src/App/App.module.css

```diff
@@ -1,32 +1,25 @@
 .wrap {
   margin: auto;
-  width: 302px;
+  width: 340px;
   font-family: 'Montserrat', sans-serif;
+  background-color: rgba(0,0,0,0.03);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5838745)